contrib/symlink-githooks.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+remove_symlinks() {
+	for hook in githooks/*
+	do
+		bn=$(basename "$hook")
+		echo "Removing git hook symlink $bn"
+		rm ".git/hooks/$bn"
+	done
+	exit 0
+}
+
+add_symlinks() {
+	mkdir -p .git/hooks
+	for hook in githooks/*
+	do
+		bn=$(basename "$hook")
+		if [ ! -f ".git/hooks/$bn" ]; then
+			echo "Symlinking $bn..."
+			ln -s "../../githooks/$bn" ".git/hooks/$bn"
+		else
+			echo "$bn hook already found, skipping"
+		fi
+	done
+}
```

Comment:
Shouldn't this also `exit 0` as well?

Reply:
It's the last executed thing so it's implied, but I can add it if we want to be explicit

Reply:
I think it is best. Because it is more "future-proof" when someone else wants to change this.

Reply:
It is updated.